cmd/cmd.go

```diff
@@ -17,8 +18,12 @@ func isTest(path string) bool {
 
 	file := filepath.Base(path)
 	name := file[:len(file)-len(ext)]
-	if name == "test" {
+	if name == "test" || name == "spec" {
```

Comment:
perl doesn't have test or spec in the name, for what it's worth. Languages that have a different pattern for the test files have this defined in the language config, so we could expose it via the API (and perhaps write it to a dotfile somewhere that we can load and check.